nlp_primitives/total_word_length.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, delimiters_regex=r"[- \[\].,!\?;\n]"):
 
     def get_function(self):
         def total_word_length(x):
-            x = x.str.len() - x.fillna(str()).str.findall(self.delimiters_regex).apply(len)
-            return x
+            delimiters = x.fillna(str()).str.findall(self.delimiters_regex)
+            counts = x.str.len() - delimiters.apply(len)
```

Comment:
Would you be able to do `x.str.count(self.delimiters_regex)` here?

I think it's slightly faster than having to findall and then check the length
![image](https://user-images.githubusercontent.com/64278226/146460023-5699aa74-230b-4fa2-afb0-c2240111cca8.png)

Reply:
yep, that works -- updated

Reply:
@tamargrey Good call!